README.md

```diff
@@ -63,8 +63,8 @@ s = shell.quote(p)
 * [common_settings](docs/common_settings_doc.md)
 * [directories](docs/copy_directory_doc.md)
```

Comment:
Can you also fix this to not be a link? `copy_directory_doc.md` is clearly an error and I don't see any page corresponding to this node.

Reply:
This would be https://github.com/bazelbuild/bazel-skylib/blob/main/docs/copy_directory_doc.md

Reply:
Right, that's linked to below in the bullet point on line 68. But on line 64, the parent item for these sub-bullets, it doesn't seem like `directories` should be a link to anything.

Reply:
Right, done.